eip155/caip168.md

```diff
@@ -55,17 +81,39 @@ Example 2
   txType: 'f(bytes32)'
 }
 ```
-**Verification Steps**
-
-1) Resolve blockchain transaction payload by `txHash` CID and `chainId`. CID is expected to be dag-eth encoded block. 
-2) Get the `data` paramater of the transaction payload, it is expected to be a hex encoded 64 byte string.
-3) The first 4 bytes are the function signature and can be discarded, the next 32 bytes is the first argument of the function, which is expected to be a 32 byte hex encoded partial CID.
-4) The partial CID is the multihash portion of the original CID. It does not include the multibase, the CID version or the IPLD codec. It is assumed that the IPLD codec is dag-cbor. Verify that the `root` CID of the timestamp anchor proof is equivalent to the transaction partial CID. Equivalence should be checked using a CIDs library, multihash portions encoded in bytes can be compared. If they are not equivalent, an error MUST be raised. 
-5) Determine that the transaction has been included in a valid block on the blockchain referenced by the `chainId`. Number of block confirmations for acceptance is up to an implementation. 
 
+#### Verification Steps
+
+1) Resolve blockchain transaction payload by `txHash` CID and `chainId`. CID is
+   expected to be [DAG-ETH][]-encoded block, and `chainId` MUST be a valid
+   [CAIP-2][] identifier.
+2) Get the `data` paramater of the transaction payload, it is expected to be a
+   hex encoded 64 byte string.
+3) The first 4 bytes are the function signature and can be discarded, the next
+   32 bytes is the first argument of the function, which is expected to be a 32
+   byte hex encoded partial CID.
+4) The partial CID is the **multihash portion** of the original CIDv1 (refer the
+   section [How does it
+   work?](https://github.com/multiformats/cid#how-does-it-work) in the
+   [Multiformats CID][] specification for context). It does not include the
+   multibase, the CID version or the IPLD codec segments. It is assumed that the
+   IPLD codec is dag-cbor. Verify that the first 32bytes of the `root` CID of
+   the timestamp anchor proof is equivalent to the 32-byte partial CID extracted
+   from the transaction data. Equivalence should be checked using a CIDs
+   library, which can compare oprtions of multihashes encoded in bytes. If the
+   two partial multihashes are not equivalent, verification MUST fail.
```

Comment:
I may have confused wording a bit using `multihash` and `partial cid` interchangeably, but it is technically just the multihash being used, so updated below to only reference that 

```suggestion
   IPLD codec is dag-cbor. Verify that the multihash portion of the `root` CID of
   the timestamp proof is equivalent to the multihash extracted
   from the transaction data. Equivalence should be checked using a CIDs
   library, which can compare multihash portions encoded in bytes. If the
   two multihashes are not equivalent, verification MUST fail.
```